packages/cli/openapi-parser/src/openapi/v3/extensions/getFernExamples.ts

```diff
@@ -55,4 +56,36 @@ export function getFernExamples(operationObject: OpenAPIV3.OperationObject): End
             description: undefined
         })
     );
+
+    const redoclyCodeSamplesKebabCase =
+        getExtension<RawSchemas.XCodeSampleSchema[]>(operationObject, OpenAPIExtension.CODE_SAMPLES) ?? [];
+    const redoclyCodeSamplesCamelCase =
+        getExtension<RawSchemas.XCodeSampleSchema[]>(operationObject, OpenAPIExtension.CODESAMPLES) ?? [];
+    const redoclyCodeSamples = [...redoclyCodeSamplesKebabCase, ...redoclyCodeSamplesCamelCase].map((value) =>
+        RawSchemas.XCodeSampleSchema.parse(value)
```

Comment:
what happens if parse fails? do we hard fail?

Reply:
pretty sure this hard fails

Reply:
don't think we want that so we should catch and log a warning using the context logger

Reply:
updated the pr with getExtensionAndValidate